### ISSUE TITLE
Animated gifs on flash

### ISSUE THREAD

**Issue by ianharrigan** [fbricker/haxe-gif#6]
Is there an example of using this lib on flash? I have gotten it to work (sort of). But it only seems to play a frame or two and then stops.

Ian

**Comment by duoguo** [comment]
+1 I have this problem too

**Comment by ianharrigan** [comment]
FWIW, there is this alternative lib: https://github.com/Yanrishatum/yagp, that seems to run across the board.

**Comment by ashes999** [comment]
+1 for YAGP, I used it in my production app and it performs quite well (even on my old Android phone).

**Comment by ianharrigan** [comment]
Yeah, i used it in haxeui's Image compoent (https://github.com/ianharrigan/haxeui/blob/master/haxe/ui/toolkit/controls/Image.hx), had no issues yet. Feel free to close this issue. :)

**Comment by ashes999** [comment]
@fbricker I see you closed this issue. Did you fix the original problem? At least three of us ran into it.

**Comment by fbricker** [comment]
@ashes999 You're right... The fact that Yagp solves the issue does not means haxe-gif should leave it unsolved. I'm reopening the issue now :+1:

**Comment by ashes999** [comment]
@fbricker you have better visibility in Google than YAGP. Having alternatives is always good. although you guys can probably team up or something too.
